flexget/plugins/clients/aria2.py

```diff
@@ -12,6 +15,135 @@
 logger = logger.bind(name='aria2')
 
 
+class JSONRPC(object):
+
+    RPC_ID = 'FLEXGET'
+    RPC_VERSION = '2.0'
+
+    ADDURI_METHOD = 'aria2.addUri'
+    GETGLOBALSTAT_METHOD = 'aria2.getGlobalStat'
+    ADDTORRENT_METHOD = 'aria2.addTorrent'
+    ADDMETALINK_METHOD = 'aria2.addMetalink'
+
+    def __init__(self, server, port, scheme, rpc_path, username=None, password=None, token=None):
+        if token:
+            self._token = 'token:' + token
+        else:
+            self._token = token
```

Comment:
Why isn't this set to empty string like userpass?

Reply:
both ```None``` and  ```''``` are correct, it's equivalent at [line50](https://github.com/Flexget/Flexget/pull/3515/files/587cacd355c4a4ce5d42bebd14f93200c8e56c2b#diff-86d45d970d5f22e4eb4842061c85f66e512d96bc34a8de8649e1ea4f10f4fe9dR50)
do I need to change it to an empty string?

Reply:
I think it would be clearer and more consistent.

Reply:
updated https://github.com/Flexget/Flexget/pull/3515/commits/aa86ea9f0c229fafac8c4756dec123b17a382742